assembl/static2/js/app/components/debate/common/headerActions.jsx

```diff
@@ -0,0 +1,46 @@
+// @flow
+import React from 'react';
+import { connect } from 'react-redux';
+import { Translate } from 'react-redux-i18n';
+import { openShareModal } from '../../../utils/utilityManager';
+
+type Props = {
+  routerParams: Map<string>,
```

Comment:
I think I added a flow type for router params (in `flow/object_types.js`). Also note that here you use `Map` as a type but you don't import it from immutable so your type is not right.